pkg/errors/conversion.go

```diff
@@ -20,12 +20,15 @@ import (
 	"errors"
 	"net"
 	"net/url"
+	"os"
 
 	"github.com/golang/protobuf/proto"
 	"google.golang.org/grpc/status"
 )
 
 var (
+	errDeadlineExceeded = DefineDeadlineExceeded("deadline_exceeded", "deadline exceeded")
```

Comment:
Can't we just treat `os.ErrDeadlineExceeded` as `context.DeadlineExceeded`?

Reply:
It also applies to things like read / writes timeout (caused by `SetReadDeadline` and so forth), but I'm ok with reusing the definition.